content/blog/findings-when-coverting-tests-to-pester5.md

```diff
@@ -0,0 +1,303 @@
+---
+title: "Findings when converting tests to Pester 5"
+date: 2020-05-17T05:00:00+01:00
+type: "post"
+draft: false
+author: johlju
+---
+
+If you have any questions or suggestions around this blog post then please
+reach out to `@johlju` in the Virtual PowerShell User Group [#DSC channel](https://dsccommunity.org/community/contact/).
+
+## Pester
+
+Pester is used throughout the DSC modules and their common modules to ensure
+that we maintain quality for all the contributions by both testing the new
+contribution (the tests added or changed in the contribution) and the
```

Comment:
you have a repeat of the word Contribution, and one is singular and the other plural.
Suggestion:
"...we maintain quality for all new contributions by testing any addition and verify they work with the rest of the code to avoid regression."